src/agentscope/studio/static/html-drag-components/agent-customizedagent.html

```diff
@@ -0,0 +1,34 @@
+<div>
+    <div class="title-box" data-class="{{agent._cls_name}}">
+        <div class="title-box-left-items">
+            <svg class="title-box-svg" viewBox="0 0 1024 1024"
+                 xmlns="http://www.w3.org/2000/svg">
+                <path d="M21.344 448h85.344v234.656H21.344V448zM554.656 192h64V106.656h-213.344V192h64v42.656h-320V896h725.344V234.656h-320V192z m234.688 128v490.656H234.688V320h554.656zM917.344 448h85.344v234.656h-85.344V448z"></path>
+                <path d="M341.344 512H448v106.656h-106.656V512zM576 512h106.656v106.656H576V512z"></path>
+            </svg>
+            <span>{{agent._cls_name}}</span>
+        </div>
+        <button class="button copy-button">Copy</button>
+        <span class="toggle-arrow">&#x25B2;</span>
+    </div>
+    <div class="box">
+        <div class="readme">
+            {{agent._cls_name}}
+            <div>Node ID: <span class="node-id">ID_PLACEHOLDER</span></div>
+        </div>
+
+        {% for key, value in agent.items() %}
+        {% if key != '_cls_name' %}
+        {% if value.type_hint == 'str' %}
+        <label>{{key}}</label>
+        <input type="text" df-args-{{key}} placeholder="{{value.default}}" data-required="true">
```

Comment:
When the key is customized or just "sys_prompt", it should be allowed to use Enter. Using <textarea> may be a better way.